FirebaseStorageSwift/Sources/SwiftAPIExtension.swift

```diff
@@ -35,64 +35,56 @@ private func getResultCallback<T>(
-
-   - Parameters:
-     - maxSize: The maximum size in bytes to download.
-     - completion: A completion block returning a Result enum with either a Data object or
-                   an NSError.
+  ///  Asynchronously downloads the object at the FIRStorageReference to a Data object.
+  ///  A Data of the provided max size will be allocated, so ensure that the device has enough
+  ///  memory to complete. For downloading large files, writeToFile may be a better option.
 
-   - Returns: A StorageDownloadTask that can be used to monitor or manage the download.
-   */
+  ///  - Parameters:
+  ///    - maxSize: The maximum size in bytes to download.
+  ///    - completion: A completion block returning a Result enum with either a Data object or
+  ///                  an NSError.
+  ///
+  ///  - Returns: A StorageDownloadTask that can be used to monitor or manage the download.
   func getData(maxSize: Int64, completion: @escaping (Result<Data, Error>) -> Void)
     -> StorageDownloadTask {
     return getData(maxSize: maxSize, completion: getResultCallback(completion: completion))
   }
 
-  /**
-   Retrieves metadata associated with an object at the current path.
-
-   - Parameters:
-     - completion: A completion block which returns a Result enum with either the
-                   object metadata or an NSError.
-   */
+  ///  Retrieves metadata associated with an object at the current path.
+  ///
+  ///  - Parameters:
+  ///   - completion: A completion block which returns a Result enum with either the
+  ///                 object metadata or an NSError.
   func getMetadata(completion: @escaping (Result<StorageMetadata, Error>) -> Void) {
     getMetadata(completion: getResultCallback(completion: completion))
   }
 
-  /**
-   Resumes a previous `list` call, starting after a pagination token.
-   Returns the next set of items (files) and prefixes (folders) under this StorageReference.
-
-   "/" is treated as a path delimiter. Firebase Storage does not support unsupported object
-   paths that end with "/" or contain two consecutive "/"s. All invalid objects in GCS will be
-   filtered.
-
-   Only available for projects using Firebase Rules Version 2.
-
-   - Parameters:
-     - withMaxResults The maximum number of results to return in a single page. Must be
-                      greater than 0 and at most 1000.
-     - pageToken A page token from a previous call to list.
-     - completion A completion handler that will be invoked with the next items and
-                  prefixes under the current StorageReference. It returns a Result enum
-                  with either the list or an NSError.
-   */
+  ///  Resumes a previous `list` call, starting after a pagination token.
+  ///  Returns the next set of items (files) and prefixes (folders) under this StorageReference.
+  ///
+  /// "/" is treated as a path delimiter. Firebase Storage does not support unsupported object
```

Comment:
nit: two spaces after `///`

Reply:
@morganchen12 Should it be one space or two? It looks like Firestore has one?

Reply:
I said two here because it seemed to be the most common across this header file. We haven't standardized; using one to be consistent with FirestoreSwift makes sense.

Reply:
As far as I'm aware, there should be one space between the three slashes and any text at the outer-most indentation level. This is what we did in `FirebaseFirestoreSwift`.

I can't say I really care too much, but at least let's be consistent. In this file there are one space, two space, and three leading space variations.